__tests__/inputs/clearlydefined.test.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation and others.
+// SPDX-License-Identifier: Apache-2.0
+
+import ClearlyDefinedSource from '../../inputs/clearlydefined';
+import nock from 'nock';
+
+test('should get package by key with after list call', async () => {
```

Comment:
I'll say this again: you are much more diligent about testing than I am. Thanks for adding these. :)